load_balancers.go

```diff
@@ -6,10 +6,13 @@ import (
 	"net/http"
 )
 
-const loadBalancersBasePath = "/v2/load_balancers"
-const forwardingRulesPath = "forwarding_rules"
-
-const dropletsPath = "droplets"
+const (
+	dropletsPath             = "droplets"
+	forwardingRulesPath      = "forwarding_rules"
+	loadBalancersBasePath    = "/v2/load_balancers"
+	loadBalancerTypeGlobal   = "GLOBAL"
+	loadBalancerTypeRegional = "REGIONAL"
```

Comment:
```suggestion
)

// Load balancer types.
const (
	LoadBalancerTypeGlobal   = "GLOBAL"
	LoadBalancerTypeRegional = "REGIONAL"
```

Reply:
Omitting `Type` field defaults to `Regional`, hence existing functionalities are preserved.